sdk-actors/src/test/java/io/dapr/actors/it/actors/ActivationDeactiviationIT.java

```diff
@@ -0,0 +1,101 @@
+package io.dapr.actors.it.actors;
+
+import io.dapr.actors.ActorId;
+import io.dapr.actors.client.ActorProxy;
+import io.dapr.actors.client.ActorProxyBuilder;
+import io.dapr.actors.it.BaseIT;
+import io.dapr.actors.it.DaprIntegrationTestingRunner;
+import io.dapr.actors.it.services.springboot.ActorService;
+import io.dapr.actors.it.services.springboot.EmptyService;
+import io.dapr.client.DaprClient;
+import io.dapr.client.DaprClientBuilder;
+import io.dapr.client.DefaultObjectSerializer;
+import io.dapr.client.domain.Verb;
+import org.junit.*;
+import org.junit.contrib.java.lang.system.EnvironmentVariables;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.junit.Assert.*;
+
+public class ActivationDeactiviationIT extends BaseIT {
+
+  private static Logger logger = LoggerFactory.getLogger(ActivationDeactiviationIT.class);
+
+  private static final AtomicInteger atomicInteger = new AtomicInteger(1);
+
+  private final String BASE_URL = "actors/%s/%s";
+
+  private final DefaultObjectSerializer serializer = new DefaultObjectSerializer();
+
+  private DaprIntegrationTestingRunner clientDaprIntegrationTestingRunner;
+
+  @Rule
+  public final EnvironmentVariables environmentVariables = new EnvironmentVariables();
+
+  @After
+  public void cleanUpTestCase() {
+    Optional.ofNullable(clientDaprIntegrationTestingRunner).ifPresent(daprRunner ->  daprRunner.destroyDapr());
+  }
+
+  @BeforeClass
+  public static void init() throws Exception {
+    daprIntegrationTestingRunner =
+        createDaprIntegrationTestingRunner(
+            "dapr initialized",
+            ActorService.class,
+            true,
+            true,
+            true,
+            30000,
+            false
+        );
+    daprIntegrationTestingRunner.initializeDapr();
+  }
+
+  @Test
+  public void testThatWhenInvokingMethodActorActivatesItselfAndDeactivesIteselfAfterElepsedTime() throws Exception {
+    Thread.sleep(20000);
+    assertTrue("Service App did not started sucessfully", daprIntegrationTestingRunner.isAppRanOK());
+    clientDaprIntegrationTestingRunner =
+        createDaprIntegrationTestingRunner(
+            "BUILD SUCCESS",
+            EmptyService.class,
+            false,
+            false,
+            true,
+            20000,
+            true
+        );
+    clientDaprIntegrationTestingRunner.initializeDapr();
+    environmentVariables.set("DAPR_HTTP_PORT", String.valueOf(clientDaprIntegrationTestingRunner.DAPR_FREEPORTS.getHttpPort()));
+    final AtomicInteger atomicInteger = new AtomicInteger(1);
+    String actorType = "DemoActorTest";
+    DefaultObjectSerializer serializer = new DefaultObjectSerializer();
+    logger.debug("Creating proxy builder");
```

Comment:
Adding a new line between the "phases" here (it looks like they're separated by logging today) would help readability.